domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt

```diff
@@ -634,6 +635,25 @@ class TopicControllerTest {
     assertThat(reviewCardResult.getOrThrow().pageContents.html).isEqualTo("<p>Description of subtopic is here.</p>")
   }
 
+  @Test
+  fun testRetrieveSubtopicTopic_validSubtopic_returnsSubtopicWithThumbnail() {
+    val topicLiveData = topicController.getTopic(FRACTIONS_TOPIC_ID)
+
+    val topic = topicLiveData.value!!.getOrThrow()
+    assertThat(topic.subtopicList.get(0).subtopicThumbnail.thumbnailGraphic).isEqualTo(SubtopicThumbnailGraphic.WHAT_IS_A_FRACTION)
+  }
+
+  @Test
+  fun testRetrieveSubtopicTopic_validSubtopic_subtopicsHaveNoThumbnailUrls() {
+    val topicLiveData = topicController.getTopic(FRACTIONS_TOPIC_ID)
+
+    // This test is intentionally verifying that there are no thumbnails for subtopics, since there are not yet any to
+    // populate.
```

Comment:
You can remove this comment as your test name is self explanatory. Also, this is good that you are writing this test case.

Reply:
Done.